engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/details/views/subscription-content-hosts.html

```diff
@@ -1,62 +1,59 @@
+  <div data-block="header">
+    <h2 translate>Attached to Hosts</h2>
   </div>
 
-  <table class="table table-bordered table-striped" bst-table="table" ng-show="contentHosts.length > 0" ng-class="{'table-mask': working}">
-    <thead>
-      <tr bst-table-head>
-        <th bst-table-column="name" sortable><span translate>Name</span></th>
-        <th bst-table-column="status">
-          {{ "Subscription Status" | translate }}
-        </th>
-        <th bst-table-column="virtual"><span translate>Virtual</span></th>
-        <th bst-table-column="environment" sortable><span translate>Environment</span></th>
-        <th bst-table-column="contentView"><span translate>Content View</span></th>
-        <th bst-table-column="serviceLevel"><span translate>Service Level</span></th>
-        <th bst-table-column="sockets"><span translate>Sockets</span></th>
-        <th bst-table-column="cores"><span translate>Cores per Socket</span></th>
-        <th bst-table-column="memory"><span translate>RAM (GB)</span></th>
-      </tr>
-    </thead>
+  <span data-block="no-rows-message" translate>
+    You currently don't have any Content Hosts using this subscription.
+  </span>
 
-    <tbody>
-      <tr ng-show="working">
-        <td colspan="9">
-          <div class="loading-mask fa-3x" ng-show="working">
-            <i class="fa fa-spinner fa-spin"></i>
-            {{ "Loading..." | translate }}
-          </div>
-        </td>
-      </tr>
+  <span data-block="no-search-results-message" translate>
+    Your search returned zero Content Hosts.
+  </span>
 
-      <tr bst-table-row ng-repeat="contentHost in contentHosts"
-          ng-controller="ContentHostStatusController">
-        <td bst-table-cell>
-          <a ui-sref="content-host.info({hostId: contentHost.host_id})">
-            {{ contentHost.name }}
-          </a>
-        </td>
-        <td bst-table-cell>
-          <span ng-class="getHostStatusIcon(contentHost['entitlement_status'])">
-          </span>
-        </td>
-        <td bst-table-cell>{{ virtual(contentHost.facts) | booleanToYesNo }}</td>
-        <td bst-table-cell>{{ contentHost.environment.name }}</td>
-        <td bst-table-cell>{{ contentHost.content_view.name || "" }}</td>
-        <td bst-table-cell>{{ contentHost.service_level }}</td>
-        <td bst-table-cell>{{ contentHost.facts.cpu['cpu_socket(s)'] }}</td>
-        <td bst-table-cell>{{ contentHost.facts.cpu['core(s)_per_socket'] }}</td>
-        <td bst-table-cell>{{ convertMemToGB(contentHost.facts.memory['memtotal']) }}</td>
-      </tr>
-    </tbody>
-  </table>
+  <div data-block="table">
+    <table class="table table-striped table-bordered"
+            ng-class="{'table-mask': table.working}">
+      <thead>
+        <tr bst-table-head row-select>
+          <th bst-table-column="name" sortable><span translate>Name</span></th>
+          <th bst-table-column="status">
+            {{ "Subscription Status" | translate }}
```

Comment:
This can be better written like this:

```html
<th bst-table-column="status" translate>
  Subscription Status
</th>
```

or 

```html
<th bst-table-column="status"> 
  <span translate>Subscription Status<span>
</th>
```